moderator.js

```diff
@@ -0,0 +1,72 @@
+var lang = require('./languages');
+
+exports.getScriptFromGame = function (game) {
+    return _getScriptFromDeck(game.deck);
+}
+
+exports.getScriptFromDeck = function (deck) {
+    return _getScriptFromDeck(deck);
+}
+
+function _getScriptFromDeck (deck) {
+    var script = '';
+
+    // To take note calls.
+    if(deck['Lycan']) script += ' - ' + lang.english['Lycan'] + '\n'; 
```

Comment:
[Single comment, not necessary to approve the PR] I think we should have another logic to do this, I suggest the next:
- Receive a string with the selected language.
- The `selectedLang` array should be sorted.
- Loop through the `selectedLang` array and do the check.

If we have a logic like this, we got some benefits:
- When we add other languages, we already have the logic to support them.
- DRY. We don't need to have so many `if` statements.
- DRY. We don't have to repeat this `' - '` and this `'\n'`.
- In order to add roles that are not in the deck, we could add some logic inside the loop to do this very easily.